demo/vl-accordion-met-niet-werkende-info-tile.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
ik zou het bestand (en vermoedelijk ook de testen) niet noemen naar "wat er niet werkt" maar naar wat er "zou moeten werken"

dus vl-accordion-met-inner-vl-info-tile of iets in die aard

Reply:
heb nu 2 met twee maal een accordion met daarin een info-tile. In het ene geval werkt de info-tile wel in het andere niet. Ik kan nu deze hernoemen maar dan moet ik die andere ook hernoemen maar naar wat dan?

Reply:
zie: 
![afbeelding](https://user-images.githubusercontent.com/1375279/115251305-b4dcce00-a12a-11eb-9698-9ebb886ffd96.png)

Reply:
naamgeving overal aangepast